### ISSUE TITLE
dots in chromosome names lead errors

### ISSUE THREAD

**Issue by andreasleha** [estepi/ASpli#2]
Dots in chromosome names (such as `AABR07024104.1`) lead to the error

    Error in data.frame(matrix(unlist(strsplit(jnames, "[.]")), byrow = TRUE,  : 
      row names supplied are of the wrong length

within the `.createGRangesExpJunctions` function.

This is triggered via `junctionDUreport`, e.g.

**Comment by estepi** [comment]
Hi, 
Yes. If the chromosome name  has a dot, it causes this problem
Can you avoid this notation? (For example, replacing with a "-", "_" or nothing.)
I can help with this if you need.
Cheers,
Estefi

**Comment by andreasleha** [comment]
Thanks.  I am fine.

I opened this report more as a feature request that ASpli should be able to handle this situation or at least give an explicit error earlier on in the pipeline about not being able to handle such names.

**Comment by estepi** [comment]
Yes, you're completely right. We are trying to fix it in order not to depend on annotation format.

**Comment by andreasleha** [comment]
Thanks for the immediate reaction to this issue.  Impressive
